service_unavailable_page/service_unavailable/manifest.yml

```diff
@@ -0,0 +1,4 @@
+---
+applications:
+- name: static-app-abeer
```

Comment:
what should this be @saliceti ?

Reply:
What about apply-maintenance? This would be consistent with apply-prod, etc

Reply:
Sounds good